scripts/exgdas_atmos_analysis_wdqms.sh

```diff
@@ -47,14 +47,14 @@ cd "${DATA}" || ( echo "FATAL ERROR: Unable to cd into '${DATA}', ABORT!"; exit
 # These should always be available
 cp "${COMIN}/${CNVSTAT}" .
 rc=$?
-(( rc != 0 )) && ( echo "FATAL ERROR: Unable to copy '${CNVSTAT}' from '${COMIN}', ABORT!"; exit 2 )
+(( rc != 0 )) && ( msg="FATAL ERROR: Unable to copy '${CNVSTAT}' from '${COMIN}', ABORT!"; echo $msg; err_exit $msg )
```

Comment:
```suggestion
(( rc != 0 )) && ( msg="Unable to copy '${CNVSTAT}' from '${COMIN}'"; err_exit "${msg}" )
```
`err_exit` adds the `FATAL ERROR:` and echo's the message provided by the argument `${msg}`.